cookbook/import_export/customize-import-behavior.rst

```diff
@@ -1,3 +1,52 @@
 How to Customize the Import Behavior
 ====================================
```

Comment:
i dropped this page on purpose from the master, it's a deprecated way to do it, this page can be fix on 1.3 branch